ssl/record/ssl3_record.c

```diff
@@ -711,16 +711,20 @@ int ssl3_get_record(SSL *s)
             goto end;
         }
 
+        /*
+         * Usually thisrr->length is accurate, just not when KTLS receives
+         * adjacent maximum-size packets.
```

Comment:
Not sure I understand this comment about "adjacent maximum-size packets"? Can you expand further? Is this an issue specific to Linux kernel?

Reply:
I don't know why these 0x4130 byte packets are received, I believe the kernel mangles two or more
packets into larger ones, and I was afraid that would break our RFC 6066 support since I doubt
the KTLS would implement the required length check.

Reply:
By RFC6066 support are you specifically referring to "max_fragment_length" (which is just one part of 6066)? If so then we don't support max_fragment_length in conjunction with KTLS anyway. If max_fragment_length is negotiated then we simply don't use KTLS.